charts/aspnetcore/templates/prechecks.tpl

```diff
@@ -0,0 +1,9 @@
+{{- if (.Values.autoscaling).enabled }}
+    {{- if not (.Values.autoscaling).minReplicas }}
+    {{- fail "autoscaling.minReplicas is required" }}
```

Comment:
I don't think it's possible for this not to be set because we define a default value in `values.yaml`.

Also, I got:
```
❯ helm template test . --set autoscaling.enabled=true --set autoscaling.minReplicas=1
Error: template: aspnetcore/templates/prechecks.tpl:5:15: executing "aspnetcore/templates/prechecks.tpl" at <le (.Values.autoscaling).minReplicas (.Values.podDisruptionBudget).minAvailable>: error calling le: incompatible types for comparison
```
It dosen't happen when I set the value in the yaml file but I think it would be prudent to cast to int anyway. I'll probably open an issue so make sure we do this everywhere the chart expects an int to be sure.